### PR TITLE
Delete a useless assertion

### PR DESCRIPTION
**Pull request by seanh**

### CODE REVIEW

File: tests/lms/services/exceptions_test.py [hunk 1]
```diff
@@ -200,5 +200,3 @@ def _requests_exception(**kwargs):
             response.raise_for_status()
         except requests.RequestException as err:
             return err
-
-        assert False, "We should never get here"
```
Comment (robertknight) [inline tests/lms/services/exceptions_test.py]: It looks like the original purpose of this was to provide a more obvious error if for some reason the code above failed to generate the expected exception. Is that not useful now?

Comment (seanh) [reply tests/lms/services/exceptions_test.py]: The line is never reached, so this just messes up the coverage report!

Comment (robertknight) [reply tests/lms/services/exceptions_test.py]: Can't you tell the code coverage generator to ignore that line?

Comment (seanh) [reply tests/lms/services/exceptions_test.py]: Ah, yes let's do that instead. Changed

**Pull request merged**